python/pylibcugraph/pylibcugraph/tests/test_katz_centrality.py

```diff
@@ -14,11 +14,17 @@
 import pytest
 import cupy as cp
 import numpy as np
-from pylibcugraph.experimental import (ResourceHandle,
-                                       GraphProperties,
-                                       SGGraph,
-                                       katz_centrality)
-from pylibcugraph.testing import utils
+from pylibcugraph import (ResourceHandle,
+                          GraphProperties,
```

Comment:
I just realized that GraphProperties was promoted and removed from experimental. Since it is not yet supported in the CAPI, should it be promoted?

Reply:
Per discussion with Chuck, graph properties doesn't change the behavior of the code but a few algos check and fail if they don't support for instance `is_multigraph = True`. So I think it is safe to have it promoted as it is.

Reply:
Yes, I agree, it would be different if it were a breaking change rather than a previously unsupported to supported behavior